assignment/core/src/main/java/com/assignment/core/jobs/LoggingJob.java

```diff
@@ -0,0 +1,27 @@
+package com.assignment.core.jobs;
+
+import org.apache.sling.event.jobs.Job;
+import org.apache.sling.event.jobs.consumer.JobConsumer;
+import org.osgi.service.component.annotations.Component;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * A JobConsumer implementation that handles logging jobs.
+ */
+@Component(service = JobConsumer.class, immediate = true, property = {
+        JobConsumer.PROPERTY_TOPICS + "=com/assignment/job/logging"
+})
+public class LoggingJob implements JobConsumer {
+
+    private static final Logger LOG = LoggerFactory.getLogger(LoggingJob.class);
+
+    /**
+     * Processes the given job and logs its ID.
+     */
+    @Override
+    public JobResult process(Job job) {
+        LOG.info("Executing Sling Job {}", job.getId());
```

Comment:
Use try catch in job process and always return JobResult.FAIL in case of any error or exception